### DISCUSSION TITLE
공식월드 캠핑에 물고기 들고 있는것 처럼 아이템을 착용하게끔 할 수있나요?

### DISCUSSION THREAD

**Discussion by hina9** [naverz/zepeto-studio-kor#757] [5. Q&A]
공식 월드 내 물고기 들고 움직일 수 있는 것처럼

제작하고 있는 맵 안에서도 캐릭터가 아이템을 장착하구 다닐 수 있는 방법이 있나요?

장착할 캐릭터가 키도 다르고 체형도 다르다 보니 어떻게 조정할지도 고민이네요ㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 아래 내용이 도움이 될 것 같습니다.
 
https://github.com/naverz/zepeto-studio-kor/discussions/542#discussioncomment-2467746

감사합니다.
